tests/node/rpc/messages/SubscribeVehicleDataResponseTests.js

```diff
@@ -29,12 +35,17 @@ describe('SubscribeVehicleDataResponseTests', function () {
             .setDataType(VehicleDataType.VEHICLEDATA_WINDOWSTATUS);
         const JSON_WINDOWSTATUS = this.windowStatus.getParameters();
 
+        this.climateData = new VehicleDataResult()
+          .setDataType(VehicleDataType.VEHICLEDATA_CLIMATEDATA);
+        const JSON_CLIMATEDATA = this.climateData.getParameters();
+
```

Comment:
**Optional**: This is not related to this PR, but on line 45 you can change `vehicleDataResult` to `handsOffSteering`.

Reply:
Based on previous experience, it is better to avoid changes not related to PR. Will add if PMs request this in the review.